sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -590,6 +590,25 @@ def test_one_hot_encoder_feature_names_unicode():
     assert_array_equal(['n👍me_c❤t1', 'n👍me_dat2'], feature_names)
 
 
+def test_one_hot_encoder_feature_names_drop():
+    # Assume that this is OK for manual drop, if OK for first
```

Comment:
I'd rather not assume this ;)

Reply:
I suspected not 😏

Reply:
maybe you can parametrize this with something like


```py
@pytest.mark.parametrize(
    'drop, expected_feature_names', [
    ('first', [...]),
    (['Female', 41, 'girl', 10], [...])
])

```

Reply:
I think that the "expected_feature_names" might get too long.

Reply:
I have implemented your suggestion @nicolashug. Are you able to give your approval so we can add to the release?